### PR TITLE
♻️ Update owners and documentation for working group changes

### PR DESCRIPTION
**Pull request by kristoferbaxter**
See ampproject/meta#65.

### CODE REVIEW

**Comment by kristoferbaxter** [comment]
Adding @rcebulko who implemented OWNERS to sanity check the changes.

File: extensions/OWNERS [hunk 1]
```diff
@@ -5,15 +5,15 @@
...
+        {name: 'ampproject/wg-components'},
         {name: 'aghassemi', requestReviews: false},
       ],
     },
     {
       pattern: '**/*-player.{js,md}',
       owners: [
         {name: 'alanorozco', notify: true},
-        {name: 'wassgha', notify: true},
+        {name: 'wassgha', notify: false},
```
Comment (rcebulko) [inline extensions/OWNERS:16]: `notify: false` is default, so that can be removed (unless you meant `requestReviews: false`)

**Review by rcebulko — APPROVED** [review]
Generally LGTM, and it passes all the owners validations checks

image

**Review by jridgewell — APPROVED** [review]

**Review by honeybadgerdontcare — COMMENTED** [review]
approval for validator related changes

**Pull request merged**